rules_building_block/discovery_net_share_discovery_winlog.toml

```diff
@@ -28,10 +28,10 @@ building_block_type = "default"
 query = '''
 sequence by user.name, source.port, source.ip with maxspan=15s 
  [file where event.action == "network-share-object-access-checked" and 
-  winlog.event_data.ShareName : ("\\*ADMIN$", "\\*C$") and 
+  winlog.event_data.ShareName in ("\\\\*\\ADMIN$", "\\\\*\\C$") and 
```

Comment:
@Samirbous do you think we need to include the IPC$ share here? Is the one that is generating more events in the telemetry, but Idk if it was intended to match your original query (`\\*C$`)

Reply:
(I'm converting this to a literal check as it was matching any patterns that ended with `c$`)

Reply:
$IPC is pretty noisy, intend was to match on c and amin shares.